src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1013,6 +1016,7 @@ def forward(
         return_loss: Optional[bool] = None,
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
+        return_base_image_embeds: Optional[bool] = None,
```

Comment:
Can't users obtain this from `vision_outputs`?

Reply:
They can just do `vision_outputs.last_hidden_state`, right?

Reply:
Unfortunately no. This is because `vision_outputs.last_hidden_state` is retrieved from the unmodified CLIP model. 

OWL-ViT first trains CLIP without any changes and then fine-tunes it along with the object detection head with some modifications to the base. I had to add this extra argument to ensure that we get the unmodified base embeddings.